src/components/Signup/Signup.test.js

```diff
@@ -277,7 +277,97 @@ describe('Signup', () => {
         getUtmCookie: jest.fn().mockImplementation(() => []),
       },
     };
-    const location = { search: 'test', pathname: '/signup' };
+
+    const search = '?utm_content=test-utm-content';
```

Comment:
I think this section to validate utm should be added in a separate test, right now I see this code as part of this test: `should show that the email with whitespace already exists when submit` 🤔